recipes/pdal/all/conanfile.py

```diff
@@ -0,0 +1,122 @@
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://pdal.io"
+    license = "BSD"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "with_unwind": [True, False],
+               "with_xml": [True, False],
+               "with_zstd": [True, False],
+               "with_laszip": [True, False],
+               }
+    default_options = {"shared": False,
+                       "fPIC": True,
+                       "with_unwind": False,
+                       "with_xml": True,
+                       "with_zstd": True,
+                       "with_laszip": True}
+    topics = ("conan", "pdal", "gdal")
+
+    _source_subfolder = "source_subfolder"
+
+    _cmake = None
+
+    def requirements(self):
+        # TODO package improvements:
+        # - switch from vendored arbiter (not in CCI). disabled openssl and curl are deps of arbiter
+        # - switch from vendor/nlohmann to nlohmann_json (in CCI)
+        # - evaluate dependency to boost instead of boost parts in vendor/pdalboost
+        self.requires("gdal/3.1.4")
+        self.requires("libgeotiff/1.6.0")
+        if self.options.with_xml:
+            self.requires("libxml2/2.9.10")
+        if self.options.with_zstd:
+            self.requires("zstd/1.4.5")
+        if self.options.with_laszip:
+            self.requires("laszip/3.4.3")
+        if self.options.get_safe("with_unwind"):
+            self.requires("libunwind/1.3.1")
+        # vendored nanoflann is 1.3.1 with minor code removal
+        self.requires("nanoflann/1.3.1")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        if self.settings.os not in ["Linux", "FreeBSD"]:
+            del self.options.with_unwind
+
+    def configure(self):
+        # upstream export/install targets do not work with static builds
+        if not self.options.shared:
+            raise ConanInvalidConfiguration("pdal does not support building as a static lib yet")
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < 5:
+            raise ConanInvalidConfiguration ("This compiler version is unsupported")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("PDAL-%s-src" % self.version, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        self._cmake = CMake(self)
+        self._cmake.definitions["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.get_safe("fPIC", True)
```

Comment:
```suggestion
```

Not required, self.options.fPIC is automatically defined: https://docs.conan.io/en/latest/reference/build_helpers/cmake.html?highlight=fpic#definitions